frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/TopToolbar/GenerateModelsButton.tsx

```diff
@@ -1,10 +1,11 @@
-import { Button, Spinner } from '@digdir/design-system-react';
```

Comment:
As you are changing Button to StudioButton, maybe we should change Spinner to StudioSpinner in the same go?

Reply:
I think that should be a separate issue, as there's already a lot of changes here, and it is not related to the update of the design system. `StudioSpinner` also have some CSS and other custom code in it, so this would also need to be tested separately.